core/templates/pages/exploration-editor-page/translation-tab/services/translation-tab-active-mode.service.ts

```diff
@@ -44,6 +47,10 @@ export class TranslationTabActiveModeService {
   isVoiceoverModeActive(): boolean {
     return this.activeMode === ExplorationEditorPageConstants.VOICEOVER_MODE;
   }
+
+  get onUpdateTranslationActiveMode(): EventEmitter<string> {
+    return this._updateTranslationActiveModeEventEmitter;
+  }
```

Comment:
```suggestion
  get onUpdateTranslationActiveMode$(): Observable<string> {
    return this._updateTranslationActiveModeEventEmitter.asObservable();
  }
```
This is better than returning the subject as you cannot call emit on an observable thereby limiting the emit calls in this service.